app/lib/package/models.dart

```diff
@@ -94,6 +94,11 @@ class Package extends db.ExpandoModel {
   Version get latestDevSemanticVersion =>
       latestDevVersionKey == null ? null : Version.parse(latestDevVersion);
 
+  bool get showDevVersion {
```

Comment:
It would also be nice to call this `showPrereleaseVersion` just to keep terminology consistent.

Reply:
This opens up a few more questions:
- `showPrereleaseVersion` or `showPreReleaseVersion` (`package:semver` has `Version.isPreRelease`)
- Are we going to rename `latestDevSemanticVersion` and similar fields too?
- What about the Datastore field?

Reply:
Fair enough, we can just leave this for a rainy day.